recipes/heliopy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "heliopy" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "d1c0bf43f156ef795cab581fd8a57c2d789ba191c2909cc4b94ea3a2777efc18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
This recipe cannot be `noarch: python` if the `skip` is used. If I suggested otherwise previously, I was mistaken and I apologize for the incorrect information.

Reply:
Does this package work on python<3.6? In the `setup.py` metadata it is the only version listed. If this is the case, you'll need to use the `skip` with the appropriate selector (https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors) and remove the `noarch`.